beta/nncf/tensorflow/graph/converter.py

```diff
@@ -168,9 +170,25 @@ def convert_keras_model_to_nncf_graph(model: tf.keras.Model) -> NNCFGraph:
         nncf_graph = _get_nncf_graph_from_functional(model)
     else:
         nncf_graph = _get_nncf_graph_from_sequential(model)
-
+    _add_aux_output_nodes_to_nncf_graph(nncf_graph)
     return nncf_graph
 
+def _add_aux_output_nodes_to_nncf_graph(nncf_graph: NNCFGraph):
+    outputs = []
+    for nx_node_key, deg in nncf_graph._nx_graph.out_degree():
+        if deg == 0:
+            outputs.append(nncf_graph._nx_node_to_nncf_node(nncf_graph._nx_graph.nodes[nx_node_key]))
+
+    for num, output_node in enumerate(outputs):
+        output_aux_node_name = 'output_{}'.format(num)
+        attrs = {
+            NNCFGraph.NODE_TYPE_ATTR: NNCFGraphNodeType.OUTPUT_NODE,
+            'original_name': output_aux_node_name
+        }
+        nncf_graph.add_node(output_aux_node_name, **attrs)
+        nncf_graph.add_edge(output_node.node_name, output_aux_node_name)
```

Comment:
This edge is missing shape and in_port attributes that all other edges have. This is important for the filter pruning algo.

Reply:
Done.